src/frontend/src/tests/lib/components/hero/ContextMenu.spec.ts

```diff
@@ -1,13 +1,18 @@
-import { BTC_MAINNET_NETWORK, ICP_NETWORK } from '$env/networks.env';
+import { BTC_MAINNET_NETWORK, ICP_NETWORK } from '$env/networks/networks.env';
 import ContextMenu from '$lib/components/hero/ContextMenu.svelte';
 import { DEFAULT_ETHEREUM_NETWORK } from '$lib/constants/networks.constants';
+import {
+	TOKEN_MENU_BTC_BUTTON,
+	TOKEN_MENU_ETH_BUTTON,
+	TOKEN_MENU_IC_BUTTON
+} from '$lib/constants/test-ids.constants';
 import { mockPage } from '$tests/mocks/page.store.mock';
 import { render } from '@testing-library/svelte';
 
 describe('ContextMenu', () => {
-	const icTokenMenuButtonSelector = 'button[data-tid="ic-token-menu-button"]';
-	const ethTokenMenuButtonSelector = 'button[data-tid="eth-token-menu-button"]';
-	const btcTokenMenuButtonSelector = 'button[data-tid="btc-token-menu-button"]';
+	const icTokenMenuButtonSelector = `button[data-tid="${TOKEN_MENU_IC_BUTTON}"]`;
+	const ethTokenMenuButtonSelector = `button[data-tid="${TOKEN_MENU_ETH_BUTTON}"]`;
+	const btcTokenMenuButtonSelector = `button[data-tid="${TOKEN_MENU_BTC_BUTTON}"]`;
```

Comment:
since you hard-coded -button , just for consistency, i would say that the above should be (and then remove the test id that ends with `_BUTTON`)

```typescript
	const icTokenMenuButtonSelector = `button[data-tid="${TOKEN_MENU_IC}-button"]`;
	const ethTokenMenuButtonSelector = `button[data-tid="${TOKEN_MENU_ETH}-button"]`;
	const btcTokenMenuButtonSelector = `button[data-tid="${TOKEN_MENU_BTC}-button"]`;
```